vendor/src/github.com/docker/engine-api/types/container/config.go

```diff
@@ -24,6 +38,7 @@ type Config struct {
 	StdinOnce       bool                  // If true, close stdin after the 1 attached client disconnects.
 	Env             []string              // List of environment variable to set in the container
 	Cmd             strslice.StrSlice     // Command to run when starting the container
+	Healthcheck     *HealthConfig         `json:",omitempty"` // Command to run to check container is healthy
```

Comment:
Only 1 health check per container? Do you see a reason why someone would want more than 1?

I can think at least testing multiple listening ports.